### PR TITLE
Avi rancher catalog

### PR DESCRIPTION
**Pull request by anantpatil**

### CODE REVIEW

**Comment by anantpatil** [comment]
@prachidamle Please review the catalog.

File: templates/avi/0/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+avi:
+  image: rancher/external-lb:v0.3.0
```
Comment (rawmind0) [inline templates/avi/0/docker-compose.yml]: You need docker image v0.3.2, due to avi PR is merged in this version. 
```
-  image: rancher/external-lb:v0.3.0
+  image: rancher/external-lb:v0.3.2
```

File: templates/avi/0/rancher-compose.yml [hunk 2]
```diff
@@ -0,0 +1,52 @@
+.catalog:
+  name: Avi Vantange Platform Load Balancer
+  version: "v0.3.0-rancher1"
```
Comment (rawmind0) [inline templates/avi/0/rancher-compose.yml]: Version like docker image version??
```
-  version: "v0.3.0-rancher1"
+  version: "v0.3.2-rancher1"
```

File: templates/avi/config.yml [hunk 3]
```diff
@@ -0,0 +1,5 @@
+name: Avi Vantage Platform
+description: |
+    External LB service powered by Avi Vantage Platform
+version: v0.3.0-rancher1
```
Comment (rawmind0) [inline templates/avi/config.yml]: Version like docker image version??
```
-version: v0.3.0-rancher1
+version: v0.3.2-rancher1
```

**Comment by rawmind0** [comment]
Hi @anantpatil .. thanks so much for the PR....

Please, take a look at the changes requested. ...

**Comment by anantpatil** [comment]
@rawmind0 Thanks for the review, will update the pull request.

File: templates/avi/0/docker-compose.yml [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+  expose: 
+   - 1000
+  environment:
+    AVI_USER: ${AVI_USER}
+    AVI_PASSWORD: ${AVI_PASSWORD}
+    AVI_CONTROLLER_ADDR: ${AVI_CONTROLLER_ADDR}
+    AVI_CONTROLLER_PORT: ${AVI_CONTROLLER_PORT}
+    AVI_SSL_VERIFY: ${AVI_SSL_VERIFY}
+    AVI_CLOUD_NAME: ${AVI_CLOUD_NAME}
+    LB_TARGET_RANCHER_SUFFIX: ${LB_TARGET_RANCHER_SUFFIX}
```
Comment (rawmind0) [inline templates/avi/0/docker-compose.yml]: Needed labels to get CATTLE_URL env variable inside container....
```
  labels:
    io.rancher.container.create_agent: "true"
    io.rancher.container.agent.role: "external-lb"
```

**Comment by rawmind0** [comment]
Hi @anantpatil ...

I'm getting errors about CATTLE_URL is not found. Please take a look at the change request to solve it. 

Thanks...

File: templates/avi/0/docker-compose.yml (hunk 4 above)
Comment (anantpatil) [reply templates/avi/0/docker-compose.yml]: Ok.

**Comment by rawmind0** [comment]
Thanks @anantpatil ....

LGTM...

**Pull request merged**